examples/hello-world-python/README.md

```diff
@@ -0,0 +1,233 @@
+# Hello World Python
+
+The is a complete "hello world" Python example to show you how to use Shroud.
+A development environment (container) is provided if needed, otherwise you
+should look at the [Dockerfile](Dockerfile) and ensure that you have basic d
+dependencies installed.
+
+## Getting Started
+
+### 0. High Level Plan
+
+Since a lot of the work has been done (and is just shown in the repository) let's
+review how you would start this from scratch. When using shroud, we want to do the following
+
+1. Write your C++ script, let's call it helloworld.cpp.
+2. Write your header file for the script, helloworld.hpp.
+3. Create a shroud input file, helloworld.yaml (note that if you have an older version of shroud the extension .yml will not work as expected)
+4. Run shroud to generate your Python files.
+5. Build your Python files
+6. Test the Python library
+
+These steps wil be explained in detail below.
+
+
+### 1. Development Environment
+
+Let's first build our development container.
```

Comment:
I've never used docker. Some sort of explanation of what this is doing would be good. Or maybe a link to some internet resource about containers.
I tend to use virtual environments but that's because the Lab machines have the correct environment. But maybe list it here as an alternative.
Would this docker example work on Windows?

Reply:
I added more verbosity about why we might use it, and links to learn more. Let me know if that works!

Reply:
And yes, Docker has a client for Windows, but I can't comment how well it works because I'm not a windows user: https://docs.docker.com/docker-for-windows/install/. I've tried it on Mac, but it seems slower to me to build, and generally interact with containers. Linux is by far the best :)